hivemind/averaging/averager.py

```diff
@@ -54,8 +55,7 @@ class DecentralizedAverager(mp.Process, ServicerBase):
     :param prefix: a shared prefix for all group keys
     :param target_group_size: attempts to form groups with up to this many peers (recommended: a power of 2, e.g. 16)
     :param initial_group_bits: a string of bits ('0' and '1') that define the initial group key (bucket index)
-    :param averaging_expiration: attempt to find a group for this many seconds, otherwise try again
-      note - this expiration time only applies to looking for group, passing tensors in allreduce may take more time
```

Comment:
If it's deprecated, the `request_timeout` docstring should no longer reference it

Reply:
ya got it :)
![image](https://user-images.githubusercontent.com/3491902/139601563-de991fa0-d8d3-4800-9388-6f4c008b9565.png)


id say it looks better off with the new name as well